README.md

```diff
@@ -1 +1,50 @@
-# node-modules-core-lab
\ No newline at end of file
+# Importing fs, url and path
+
+## Objectives
+
+1. Write a script to import fs, url and path
+1. Invoke methods from core modules fs, url and path
+1. Output results
+
+## Introduction
+
+There's a log with records of HTTP requests. They look like URLs with a protocol, domain, path, etc. Your task is to create a script which will create a new file with only the paths from each URL.
+
+In this lab, you'll start mastering the three core modules: fs, url and path.
+
+## Instructions
+
+1. Create a module file `url-transformer.js`. It must export a function with three arguments: input file name, output file name and callback.
+1. In `url-transformer`, read a data file `data/urls.txt` using `fs` and `path.join()` to point to the file.
+1. Parse each line of the data from the `urls.txt` file with the `url` core module to get only the path name part, i.e., strip away the domain and protocol.
+1. Write only the path names to a new file `output/paths.txt` using `path` for the path to the file.
+1. Callback with "Transformation completed" when the job is done.
+1. Results will be tested at random so manual parsing won't work. ;-)
+1. Test your module with your CLI runner `index.js` by executing `node .` in the terminal. `node .` is an equivalent to `node index` or `node index.js`.
```

Comment:
As noted below, I'm not sure that we need to be this explicit. I think it might be better just to let students know that they can use the tools that they learned in previous lessons (about importing, exporting, etc.) to try out their code.

I do like the note about `index.js`, though.